feature/static/otg_tests/next_hop_encap_udp/static_encap_gue_ipv4_test.go

```diff
@@ -0,0 +1,1176 @@
+		dut.Port(t, "port3"),
+	}
+	configureDUTLag(t, dut, dutAggPorts1, dutLag1Name, dutLAG1)
+
+	// Ports 4 and 5 will be part of LAG
+	dutAggPorts2 := []*ondatra.Port{
+		dut.Port(t, "port4"),
+		dut.Port(t, "port5"),
+	}
+	configureDUTLag(t, dut, dutAggPorts2, dutLag2Name, dutLAG2)
+
+	root := &oc.Root{}
+	ni := root.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+
+	t.Log("Configuring BGP")
+	bgpProtocol := ni.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := bgpProtocol.GetOrCreateBgp()
+	g := bgp.GetOrCreateGlobal()
+	g.As = ygot.Uint32(dutAS)
+
+	pg := bgp.GetOrCreatePeerGroup(bgpPeerGroupName)
+	pg.PeerAs = ygot.Uint32(ateAS)
+
+	ipv6Nbr := bgp.GetOrCreateNeighbor(atePort1.IPv6)
+	ipv6Nbr.PeerGroup = ygot.String(bgpPeerGroupName)
+	ipv6Nbr.PeerAs = ygot.Uint32(ateAS)
+	ipv6Nbr.Enabled = ygot.Bool(true)
+	ipv6Nbr.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)
+
+	gnmi.Update(t, dut, d.NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Config(), bgpProtocol)
+
+	// Configure GUE Encap
+	configureGueTunnel(t, dut, gueDstIPv4, 0, 0)
+
+	t.Log("Configuring Static Routes")
+
+	// Configuring Static Route: IPv4-DST-GUE --> ATE:LAG1:IPv4 & ATE:LAG2:IPv4.
+	b := &gnmi.SetBatch{}
+	sV4 := &cfgplugins.StaticRouteCfg{
+		NetworkInstance: deviations.DefaultNetworkInstance(dut),
+		Prefix:          staticgueDstIPv4,
+		NextHops: map[string]oc.NetworkInstance_Protocol_Static_NextHop_NextHop_Union{
+			"0": oc.UnionString(ateLAG1.IPv4),
+			"1": oc.UnionString(ateLAG2.IPv4),
+		},
+	}
+
+	if _, err := cfgplugins.NewStaticRouteCfg(b, sV4, dut); err != nil {
+		t.Fatalf("Failed to configure IPv4 static route: %v", err)
+	}
+	b.Set(t, dut)
+
+	// Configuring Static Route: PNH-IPv6 --> IPv4 GUE tunnel.
+	sV4 = &cfgplugins.StaticRouteCfg{
+		NetworkInstance: deviations.DefaultNetworkInstance(dut),
+		Prefix:          staticpnhIPv6,
+		NextHops: map[string]oc.NetworkInstance_Protocol_Static_NextHop_NextHop_Union{
+			"0": oc.UnionString(nexthopGroupName),
+		},
+	}
+
+	cfgplugins.NewStaticRouteNextHopGroupCfg(t, b, sV4, dut, nexthopGroupName)
+
+	sV4 = &cfgplugins.StaticRouteCfg{
+		NetworkInstance: deviations.DefaultNetworkInstance(dut),
+		Prefix:          staticpngv6IPv6,
+		NextHops: map[string]oc.NetworkInstance_Protocol_Static_NextHop_NextHop_Union{
+			"1": oc.UnionString(nexthopGroupNameV6),
+		},
+	}
+
+	cfgplugins.NewStaticRouteNextHopGroupCfg(t, b, sV4, dut, nexthopGroupNameV6)
+
+	// Apply load balancing
+	if deviations.LoadIntervalNotSupported(dut) {
+		cli := `
+			load-balance policies
+			load-balance sand profile default
+			packet-type gue outer-ip
+			`
+		helpers.GnmiCLIConfig(t, dut, cli)
+	} else {
+		// TODO: OC does not yet support selecting the load-balancing hash mode on LAG members.
+		t.Logf("Load balancing is currently not supported via OpenConfig. Will fix once it's implemented.")
+	}
```

Comment:
This deviation is being misused.  Load interval and load balancing are two different things.  

This config looks like it is arista specific and is global device initialization style of configuration.  This should probably be moved into the dut initialization cfgplugin.